### ISSUE TITLE
Error Occurred When No Typed Email at DB When Login 

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#64]
When user gives the email which is not existed in the db at the Login page, the error is occurred like the below.

 ```
         if (passwordHash.verify(password ,rows.rows[0].password)) {
                                                         ^

TypeError: Cannot read property 'password' of undefined
    at Query.<anonymous> (/Users/junghwanyim/CSE442/CSE442-BuyNSell/server.js:57:58)
    at Query.handleReadyForQuery (/Users/junghwanyim/CSE442/CSE442-BuyNSell/node_modules/pg/lib/query.js:124:10)
    at Connection.<anonymous> (/Users/junghwanyim/CSE442/CSE442-BuyNSell/node_modules/pg/lib/client.js:173:19)
    at Connection.emit (events.js:182:13)
    at Socket.<anonymous> (/Users/junghwanyim/CSE442/CSE442-BuyNSell/node_modules/pg/lib/connection.js:136:12)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
```

**Comment by JacobYim** [comment]
It has been solved by try catch. 

```
try{
            if (passwordHash.verify(password ,rows.rows[0].password)) {
                res.send('Login Success!!');
                // res.redirect('/');
            } else {
                res.send('Login Failure');
                // res.redirect('/');
            }
          }catch(err){
            res.send('user is not exist')
          }
```

**Comment by JacobYim** [comment]
This solution merged at #65.
